packages/ChangesReloaded-Core.package/CRBrowser.class/instance/isLoadButtonEnabled.st

```diff
@@ -0,0 +1,4 @@
+buttons
+isLoadButtonEnabled
+
+	^ self numberOfPickedChangeListItems ~= 0
```

Comment:
What exactly is the meaning of ~=? I think I get what you are trying to do here but is the name of this method really what this is? shouldn't it be something like "areChangesSelected"?

Reply:
Also: I wanted to comment on just line 4 but github didn't let me </3

Reply:
This method returns true if the load button should be enabled and false of not
The load button should be enabled when there are changes picked. ~= means "unequal"